flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDialectITCase.java

```diff
@@ -1121,6 +1122,35 @@ public void testSetCommand() throws Exception {
         assertThat(result.toString()).isEqualTo(String.format("[+I[%s]]", path));
     }
 
+    @Test
+    public void testAddCommand() {
+        TableEnvironmentInternal tableEnvInternal = (TableEnvironmentInternal) tableEnv;
+        Parser parser = tableEnvInternal.getParser();
+
+        // test add jar
+        Operation operation = parser.parse("add jar test.jar").get(0);
+        assertThat(operation).isInstanceOf(AddJarOperation.class);
+        assertThat(((AddJarOperation) operation).getPath()).isEqualTo("test.jar");
+        // test add jar with variable substitute
+        operation = parser.parse("add jar \"${hiveconf:common-key}.jar\"").get(0);
+        assertThat(operation).isInstanceOf(AddJarOperation.class);
+        assertThat(((AddJarOperation) operation).getPath()).isEqualTo("\"common-val.jar\"");
```

Comment:
Why use double quotes here? IIUC, this is invalid in ADD JAR command?

Reply:
Actually this a valid `ADD JAR command` at least for parse phase.
For command: `add file "df.xf"`
In hive: it will throw exception `Query returned non-zero code: 1, cause: java.net.URISyntaxException: Illegal character in path at index 0: "df.xf"` which is actually thrown in execution phase.
In spark: it won't throw exception, and the `"` will be escaped to `%22`.

Reply:
I added a commit to remove the quotes. If we want to verify negative behavior, please add exception tests. It seems Hive doesn't support double and single quotes on the path. We can support this feature in the future if users want it.